### DISCUSSION TITLE
How to integrate Brightpearl in Laravel

### DISCUSSION THREAD

**Discussion by cerberusmaster** [cerberusmaster/OMDB-Back-End#4] [Q&A] [answered]
Hello Everyone.

who has experience in Brightpearl integration in laravel?

**Comment by FFS28** [comment, marked as answer]
I have experience in Brightpearl integration.
you can use Brightpearl APIs.
So use curl command in laravel.
You'll get response data from Brightpearl API

**Reply by cerberusmaster** [reply]
Thanks
